### PR TITLE
Vox raider day 0 fixes

### PR DESCRIPTION
**Pull request by ShiftyRail**
Made a goof in the timer display.
Made the raiders not delete themselves, but stay in the shuttle for a time before they go back as ghosts.

More to come, of course, those were just the easiest problems to identify.

### CODE REVIEW

File: code/datums/gamemode/factions/vox_shoal.dm [hunk 1]
```diff
@@ -223,6 +222,10 @@ var/list/potential_bonus_items = list(
 		else
 			results = "The vox raiders didn't beat the previous record of [score_to_beat]."
 
+		for (var/datum/role/R in members)
+			to_chat(R.antag.current, "<span class='notice'>The raid is over. You'll go back to the shoal in a few minutes...</span>")
+			spawn (1 MINUTES)
+				qdel(R.antag.current)
```
Comment (Probe1) [inline code/datums/gamemode/factions/vox_shoal.dm:228]: This is only going to effect raiders right?

Comment (ShiftyRail) [reply code/datums/gamemode/factions/vox_shoal.dm:228]: Yes.

**Comment by Irradiation** [comment]
Muh changelog

**Comment by Probe1** [comment]
Oops

**Pull request merged**